### ISSUE TITLE
proposal: optional @aws section

### ISSUE THREAD

**Issue by brianleroux** [arc-repos/arc-workflows#67]
Most AWS workflows require `AWS_PROFILE` and `AWS_REGION`. They env vars are a standard practice and a recommended practice by Amazon. They are a good idea. We need to separate our various AWS accounts and profile makes sense for that. Regions are the new servers and we rightly need to account for that. 

What we don't need to do is repeat this configuration in every single `npm run` script declaration in `package.json`. Well, we do right now, and its a bit visually jarring. 

Here's a proposed syntax addition to `.arc`:

```.arc
@app
my-kewl-app

@aws
region us-east-1
profile smallwins

@html
get /
```

This would mean a minor refactor to arc-* binaries to read the local .arc file and glean these variables (should they exist) falling back to whatever is in the local env. `npm run` scripts would override. But I do think the `package.json` files would read a whole lot nicer.

**Comment by ryanblock** [comment]
Love this idea!

**Comment by brianleroux** [comment]
implemented in 3.1 🎉
